[Bug][T2]: `test_voq_fabric_isolation.py` fails with `Failed: auto_isolated state for port 91 should be 0`

### Issue Description

```
            cmd = "sonic-db-cli {} STATE_DB hset 'FABRIC_PORT_TABLE|PORT{}' TEST_CRC_ERRORS 0".format(asicName, port)
            cmd_output = duthost.shell(cmd, module_ignore_errors=True)["stdout"].split("\n")
            cmd = "sonic-db-cli {} STATE_DB hset 'FABRIC_PORT_TABLE|PORT{}' TEST_CODE_ERRORS 0".format(asicName, port)
            cmd_output = duthost.shell(cmd, module_ignore_errors=True)["stdout"].split("\n")
            cmd = "sonic-db-cli {} STATE_DB hset 'FABRIC_PORT_TABLE|PORT{}' TEST 'TEST'".format(asicName, port)
            cmd_output = duthost.shell(cmd, module_ignore_errors=True)["stdout"].split("\n")

>           pytest_assert(wait_until(600, 60, 0, check_fabric_link_status, duthost, asicName, port, '0'),
                          "auto_isolated state for port {} should be 0".format(port))
E                         Failed: auto_isolated state for port 91 should be 0

asic       = 0
asicName   = '-n asic0'
cmd        = "sonic-db-cli -n asic0 STATE_DB hset 'FABRIC_PORT_TABLE|PORT91' TEST 'product'"
cmd_output = ['0']
duthost    = <MultiAsicSonicHost cmp214-5>
duthosts   = [<MultiAsicSonicHost cmp214-5>, <MultiAsicSonicHost cmp214-6>, <MultiAsicSonicHost cmp214>]
enum_frontend_dut_hostname = 'cmp214-5'
lnkStatus  = 'up'
num_asics  = 2
port       = 91

voq/test_voq_fabric_isolation.py:66: Failed
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> entering PDB >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>

>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> PDB post_mortem (IO-capturing turned off) >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
> /data/tests/voq/test_voq_fabric_isolation.py(66)test_voq_fabric_isolation_status()
-> pytest_assert(wait_until(600, 60, 0, check_fabric_link_status, duthost, asicName, port, '0'),
```

The problem is the `FABRIC_PORT_TABLE|PORT` doesn't have the attribute `AUTO_ISOLATED` by default, it only shows up the first time the fabric port becomes isolated:
https://github.com/sonic-net/sonic-swss/blob/202405/orchagent/fabricportsorch.cpp#L855
```
            if (autoIsolated == 0 && (consecutivePollsWithErrors >= isolationPollsCfg
                                   || consecutivePollsWithFecErrs >= fecIsolatedPolls))
            {
                // Link needs to be isolated.
                SWSS_LOG_INFO("port %s auto isolated", key.c_str());
                autoIsolated = 1;
                updateStateDbTable(m_stateTable, key, "AUTO_ISOLATED", autoIsolated);
                SWSS_LOG_NOTICE("port %s set AUTO_ISOLATED %d", key.c_str(), autoIsolated);
            }
            else if (autoIsolated == 1 && consecutivePollsWithNoErrors >= recoveryPollsCfg
                  && consecutivePollsWithNoFecErrs >= fecUnisolatePolls)
            {
                // Link is isolated, but no longer needs to be.
                SWSS_LOG_INFO("port %s healthy again", key.c_str());
                autoIsolated = 0;
                updateStateDbTable(m_stateTable, key, "AUTO_ISOLATED", autoIsolated);
                SWSS_LOG_NOTICE("port %s set AUTO_ISOLATED %d", key.c_str(), autoIsolated);
            }
```

In the test it first checks that `AUTO_ISOLATED=0` which will fail if the port hasn't even been isolated:
```
pytest_assert(wait_until(600, 60, 0, check_fabric_link_status, duthost, asicName, port, '0'),
              "auto_isolated state for port {} should be 0".format(port))
```

### Results you see

`FABRIC_PORT_TABLE|PORT` is missing the `AUTO_ISOLATED` attribute

### Results you expected to see

The test should probably just treat a `FABRIC_PORT_TABLE|PORT` entry with no `AUTO_ISOLATED` the same as `AUTO_ISOLATED=0`

### Is it platform specific

generic

### Relevant log output

_No response_

### Output of `show version`

_No response_

### Attach files (if any)

_No response_

--- a/tests/voq/test_voq_fabric_isolation.py
+++ b/tests/voq/test_voq_fabric_isolation.py
@@ -110,7 +110,8 @@ def check_fabric_link_status(host, asicName, port, state):
     if auto_isolated == state:
         return True
     elif auto_isolated == '' and state == '0':
-        # AUTO_ISOLATED attribute may be missing from the table if it's the first time it's been isolated, missing means not isolated
+        # AUTO_ISOLATED attribute may be missing from the table if it's the first time it's been isolated,
+        # missing means the port is not isolated
         return True
     else:
         return False

--- a/tests/voq/test_voq_fabric_isolation.py
+++ b/tests/voq/test_voq_fabric_isolation.py
@@ -109,5 +109,8 @@ def check_fabric_link_status(host, asicName, port, state):
     auto_isolated = cmd_output[0]
     if auto_isolated == state:
         return True
+    elif auto_isolated == '' and state == '0':
+        # AUTO_ISOLATED attribute may be missing from the table if it's the first time it's been isolated, missing means not isolated
+        return True
     else:
         return False

